modules/analysis/Cargo.toml

```diff
@@ -0,0 +1,65 @@
+[package]
+name = "trustify-module-analysis"
+version.workspace = true
+edition.workspace = true
+publish.workspace = true
+license.workspace = true
+
+[dependencies]
+trustify-auth = { workspace = true }
+trustify-common = { workspace = true }
+trustify-entity = { workspace = true }
+trustify-infrastructure = { workspace = true }
+
+actix-http = { workspace = true }
+actix-web = { workspace = true }
+anyhow = { workspace = true }
+async-graphql = { workspace = true, features = ["uuid", "time"] }
+cpe = { workspace = true }
+futures-util = { workspace = true }
+itertools = { workspace = true }
+log = { workspace = true }
+roxmltree = { workspace = true }
```

Comment:
I think @helio-frota has some cargo invocation to find "excess" dependencies?

Do you actually do XML parsing?

Reply:
not me ... maybe I picked it up at some point inadvertently ?

Reply:
I know a bit cargo bits :dark_sunglasses:

Reply:
`RUSTFLAGS=-Wunused-crate-dependencies cargo build &> unused.txt`

Reply:
some of them are not true (yeah I saw the un-true before)
https://gist.github.com/helio-frota/49ba11927a1f211a0f48b99f3e21bc24

Reply:
thx @helio-frota very useful that!